Shared.IntegrationTesting/SpecflowTableHelper.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace Shared.IntegrationTesting
+{
+    using System;
+    using TechTalk.SpecFlow;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public static class SpecflowTableHelper
+    {
+        #region Methods
+
+        /// <summary>
+        /// Gets the boolean value.
+        /// </summary>
+        /// <param name="row">The row.</param>
+        /// <param name="key">The key.</param>
+        /// <returns></returns>
+        public static Boolean GetBooleanValue(TableRow row,
+                                              String key)
+        {
+            String field = SpecflowTableHelper.GetStringRowValue(row, key);
+
+            return bool.TryParse(field, out Boolean value) && value;
+        }
+
+        /// <summary>
+        /// Gets the date for date string.
+        /// </summary>
+        /// <param name="dateString">The date string.</param>
+        /// <param name="today">The today.</param>
+        /// <returns></returns>
+        public static DateTime GetDateForDateString(String dateString,
+                                                    DateTime today)
+        {
+            switch(dateString.ToUpper())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/stuart_ferguson1/Shared/pullRequest?prid=5464232)